File: scripts/languages-wrapper/execute.py
import os 
import json
from solution import solution

current_directory = os.path.dirname(__file__)
input_path = os.path.join(current_directory, "input.json")
output_path = os.path.join(current_directory, "output.json")

with open(input_path, "r") as file_content:
    input_json = json.load(file_content)

with open(output_path, "w") as file_content:
    json.dump(solution(*input_json), file_content)

File: scripts/languages-wrapper/execute.ts
import path from 'path'
import * as fsWithCallbacks from 'fs'
// @ts-ignore
import solution from './solution'

const fs = fsWithCallbacks.promises
const inputPath = path.join(__dirname, 'input.json')
const outputPath = path.join(__dirname, 'output.json')

const main = async () => {
  const inputFile = await fs.readFile(inputPath, { encoding: 'utf-8' })
  const inputJSON = JSON.parse(inputFile.toString())

  try {
    const result = solution.apply(null, inputJSON)
    await fs.writeFile(outputPath, JSON.stringify(result), { encoding: 'utf-8' })
  } catch (error) {
    console.error(error)
  }
}

main()
